demos/face_detection_mtcnn_demo/python/face_detection_mtcnn_demo.py

```diff
@@ -0,0 +1,308 @@
+
+    model_proposal = models.ProposalModel(ie, args.model_proposal)
+    model_refine = models.RefineModel(ie, args.model_refine)
+    model_output = models.OutputModel(ie, args.model_output)
+
+    detector_pipeline = MtcnnPipeline(ie, model_proposal, model_refine, model_output,
+                                      pm_sync=not args.proposal_async,
+                                      pm_device=args.device_proposal,
+                                      rm_batch_size=args.refine_batch_size,
+                                      rm_config=refine_config_plugin,
+                                      rm_num_requests=args.refine_requests,
+                                      rm_device=args.device_refine,
+                                      om_batch_size=args.output_batch_size,
+                                      om_config=output_config_plugin,
+                                      om_num_requests=args.output_requests,
+                                      om_device=args.device_output)
+
+    cap = open_images_capture(args.input, args.loop)
+
+    log.info('Starting inference...')
+    print("Use 'c' key to disable/enable confidence drawing, 'l' to disable/enable landmarks drawing")
+    print("To close the application, press 'CTRL+C' here or switch to the output window and press ESC key")
+
+    palette = ColorPalette(1)
+    metrics = PerformanceMetrics()
+    video_writer = cv2.VideoWriter()
+
+    draw_lanmdmark = True
+    draw_confidence = True
+    total_frames = 0
+    while True:
+        start_time = perf_counter()
+        frame = cap.read()
+        if not frame:
+            break
+        total_frames += 1
+        if total_frames == 1 :
+            presenter = monitors.Presenter(args.utilization_monitors, 55,
+                                           (round(frame.shape[1] / 4), round(frame.shape[0] / 8)))
+            if args.output:
+                video_writer = cv2.VideoWriter(args.output, cv2.VideoWriter_fourcc(*'MJPG'), cap.fps(),
+                                               (frame.shape[1], frame.shape[0]))
+                if not video_writer.isOpened():
+                    raise RuntimeError("Can't open video writer")
+        detections = detector_pipeline.infer(frame)
+
+        presenter.drawGraphs(frame)
+        draw_detections(frame, detections, palette, None, 0.5, draw_lanmdmark, draw_confidence)
+        metrics.update(start_time, frame)
+
+        if video_writer.isOpened() and (args.output_limit == -1 or total_frames <= args.output_limit - 1):
+            video_writer.write(frame)
+
+        if not args.no_show:
+            cv2.imshow('Detection Results', frame)
+            key = cv2.waitKey(1)
+            ESC_KEY = 27
+            # Quit.
+            if key in {ord('q'), ord('Q'), ESC_KEY}:
+                break
+            if key in {ord('l'), ord('L')}:
+                draw_lanmdmark = not draw_lanmdmark
+            if key in {ord('c'), ord('C')}:
```

Comment:
```suggestion
            elif key in {ord('c'), ord('C')}:
```
And let `presenter` handle what's left. BTW `c` key is reserved for `presenter` to toggle CPU load. And I don't see why you need to flip `draw_confidence` so I suggest to remove it.

Reply:
That was done for case with many face on the frame.  But this option can easily be removed.

Reply:
Done - reassigned to `s` key. Ready to remove if necessary.
@vladimir-dudnik what do you think about turning on/off landmarks and scores drawing? On human-dense image it could be vital for boxes recognition.